test/integration/DCAHubCompanion/multi-call.spec.ts

```diff
@@ -461,6 +578,61 @@ contract('Multicall', () => {
     return data!;
   }
 
+  async function withdrawSwappedData({ positionId, recipient }: { positionId: BigNumberish; recipient: HasAddress }) {
+    const { data } = await DCAHubCompanion.populateTransaction.withdrawSwapped(DCAHub.address, positionId, recipient.address);
+    return data!;
+  }
+
+  async function reduceAllPositionData({ positionId, recipient }: { positionId: BigNumberish; recipient: HasAddress }) {
+    const { remaining } = await DCAHub.userPosition(positionId);
+    const { data } = await DCAHubCompanion.populateTransaction.reducePosition(DCAHub.address, positionId, remaining, 0, recipient.address);
+    return data!;
+  }
+
+  async function depositAllInCompanionData({ from, to }: { from: IERC20; to: IERC20 }) {
+    const { data } = await DCAHubCompanion.populateTransaction.depositWithBalanceOnContract(
+      DCAHub.address,
+      from.address,
+      to.address,
+      1,
+      SwapInterval.ONE_DAY.seconds,
+      positionOwner.address,
+      [],
+      []
+    );
+    return data!;
+  }
+
+  async function depositFromCompanionData({ from, to, amount }: { from: IERC20; to: IERC20; amount: BigNumberish }) {
+    const { data } = await DCAHubCompanion.populateTransaction.deposit(
+      DCAHub.address,
+      from.address,
+      to.address,
+      amount,
+      1,
+      SwapInterval.ONE_DAY.seconds,
+      positionOwner.address,
+      [],
+      []
+    );
+    return data!;
+  }
+
+  async function takeFromCallerData({ token, amount }: { token: IERC20; amount: BigNumberish }) {
+    const { data } = await DCAHubCompanion.populateTransaction.takeFromCaller(token.address, amount);
+    return data!;
+  }
+
+  async function sendToRecipient({ token, amount, recipient }: { token: IERC20; amount: BigNumberish; recipient: HasAddress }) {
+    const { data } = await DCAHubCompanion.populateTransaction.sendToRecipient(token.address, amount, recipient.address);
+    return data!;
+  }
+
+  async function sendAllInCompanionToRecipient({ token, recipient }: { token: IERC20; recipient: HasAddress }) {
+    const { data } = await DCAHubCompanion.populateTransaction.sendBalanceOnContractToRecipient(token.address, recipient.address);
+    return data!;
+  }
```

Comment:
This two don't have the `Data` suffix, I really don't mind it, but maybe for consistency?

Reply:
Added!